components/TARGET_PSA/services/crypto/COMPONENT_SPE/crypto_partition.c

```diff
@@ -0,0 +1,1376 @@
+// ---------------------------------- Includes ---------------------------------
```

Comment:
i cannot review the changes in this file. It was renamed and modified and now appears as a new file.
can we split rename and modify commits so the changes will be reviewable?